src/Tribe/REST/V1/Post_Repository.php

```diff
@@ -962,6 +962,11 @@ protected function build_attendee_data( array $attendee, $context = 'default' )
 					'email'           => Tribe__Utils__Array::get( $attendee, 'holder_email', Tribe__Utils__Array::get( $attendee, 'purchaser_email', '' ) ),
 					'checked_in'      => $checked_in,
 					'checkin_details' => $checkin_details,
+
+					// Show Attendee flags.
+					// @todo Make these live in future IAC work.
+					'is_subscribed' => false,
+					'is_purchaser'  => true,
```

Comment:
:warning: Array double arrow not aligned correctly; expected 4 space(s) between "'is_purchaser'" and double arrow, but found 2.

<sub>hash: 769f38b9479112a21bafcf40f788c767</sub>